.travis.yml

```diff
@@ -7,17 +7,16 @@ services:
 jobs:
   include:
     # dont push to dockerhub on forks
-    # TODO: restore integration tests on Travis once the story around pushing flyteadmin images to dockerhub is resolved
     - if: fork = true
       stage: test
       name: build test
       install: true
       script: BUILD_PHASE=builder make docker_build
     - if: fork = false
       stage: test
-      name: build
+      name: build, integration test, and push
       install: true
-      script: BUILD_PHASE=builder make docker_build
+      script: BUILD_PHASE=builder make docker_build && make k8s_integration && make dockerhub_push
```

Comment:
but we'll get rid of this right?

Reply:
no, why?

Reply:
i thought we didn't want to push to dockerhub on PRs

Reply:
this is if `fork = false`

Reply:
oh that's what that does?  cool

Reply:
ugh forget it, i'll figure out creds to push in another pr